crates/flox-rust-sdk/src/providers/git.rs

```diff
@@ -434,6 +434,20 @@ impl GitCommandProvider {
         Ok(())
     }
 
+    pub fn push2(
```

Comment:
blocker: surely there's a more informative name for this method? At the very least can you add a doc comment?

Reply:
yes, there was a `push` method in the trait already that i had to work around. will make this clearer.